drivers/misc/ethos_u/init.c

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (c) 2021-2022 Arm Limited.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "ethosu_driver.h"
+#include <device.h>
+#include <devicetree.h>
+#include <init.h>
+#include <kernel.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/printk.h>
+#include <sys/util.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_DECLARE(ethos_u, CONFIG_KERNEL_LOG_LEVEL);
```

Comment:
Shouldn't this be `LOG_MODULE_REGISTER` as this is the only log module for ethos